homeassistant/components/awair/translations/en.json

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
Was this file meant to be committed?

Reply:
It doesn't really matter. It's the output of `python3 -m script.translations develop`

Reply:
It will be overridden once translations update at night.